release/sdk-config.yaml.tmpl

```diff
@@ -50,6 +50,12 @@ commands:
   path: daml-helper/daml-helper
   desc: "Interact with a DAML ledger (experimental)"
   args: ["ledger"]
+- name: codegen
+  path: daml-helper/daml-helper
+  desc: "Run a language bindings code generation tool"
```

Comment:
I would suggest to not have a description here until this is a bit better documented (iirc the current docs still don’t tell me how to actually invoke it) so it doesn’t show up in `--help`. If we do really want for it to show up in `--help` we should at least add `(experimental)` like we do for other commands.

Reply:
The docs for TypeScript will tomorrow. The existing desc. has been there since time immemorial. Are you sure you want me to hide it?

Reply:
Ah I forgot that this is also the codegen for Java and Scala. In that case, let’s keep it.

Reply:
K, k.